report/report.tex

```diff
@@ -134,17 +134,18 @@ \subsection{Data Preprocessing}
 Each review was then tokenized into separate words using whitespace as a delimiter, as part of lemmatization to reduce morphological variation. Shortening some words to their root was thought important to improve model training efficiency. Lemmatization was chosen over stemming, as the accuracy of a lemmatizer in properly shortening words was more important than the speed of a stemmer.
 
 As the final step of our data cleanup a column “voteSuccess” was added to the dataframe, calculating helpfulness of a review by dividing number of votes from the “votes”-column by quarters elapsed since the review was submitted. This was done to create a fairer indicator of helpfulness, since newer reviews would have had less time to accumulate helpfulness-votes and therefore appear less helpful.
+
+\subsection{Ethics and bias}
+When you consider ethics in NLP, you need to first look at the bias the can occur. A review might say “Great product, used it for a week!”. This is perceived as helpful by some customers, since the reviewer expresses their opinion on the product with sentiment information. But some other consumer might not find it helpful due to the lack of more details or an analysis of the product. This leads to reviews being biased towards the consumers uses, and needs.
```

Comment:
Ville nok tilføje noget i stil med: "This can make helpfulness an unstable metric when measuring the value of a review." til allersidst, i første afsnit.